### PR TITLE
Export Tags When Exporting Beneficiaries from Manage Beneficiaries

### PR DESCRIPTION
**Pull request by vahidbazzaz**
This PR adds tags to exports. Additionally, when the number of beneficiaries exceeds 500, an "Export All" button is displayed to export all beneficiaries.

Screenshot 2024-12-13 at 2 56 21 PM

### CODE REVIEW

**Comment by vahidbazzaz** [comment]
[Dropapp: export tags when exporting beneficiaries from Manage Beneficiaries](https://trello.com/c/TYgsqCRH/1568-dropapp-export-tags-when-exporting-beneficiaries-from-manage-beneficiaries)

**Review by HaGuesto — APPROVED** [review]
@vahidbazzaz I refactored your code a bit, but great PR.

File: include/people_export.php [hunk 1]
```diff
@@ -1,24 +1,55 @@
...
+    SELECT 
+        p.*, 
+        DATE_FORMAT(FROM_DAYS(DATEDIFF(NOW(), p.date_of_birth)), "%Y")+0 AS age, 
+        IF(ISNULL(p.parent_id), p.id, p.parent_id) AS boxtribute_family_id, 
+        IF(ISNULL(p.parent_id), TRUE, FALSE) AS familyhead,
+        (
+            SELECT GROUP_CONCAT(t.label) 
+            FROM tags t 
+            INNER JOIN tags_relations tr ON t.id = tr.tag_id 
+            WHERE tr.object_type = "People" AND tr.object_id = p.id
```
Comment (pylipp) [inline include/people_export.php:32]: This will need `AND tr.deleted_on IS NULL`

Comment (pylipp) [reply include/people_export.php:32]: cf. #649

**Pull request merged**